tsconfig.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "allowJs": true,
```

Comment:
Technically, I think this setting can be removed now.  It was only there while the codebase was being converted, but now everything should be TS going forward.